AutomatedTesting/Gem/PythonTests/editor/TestSuite_Main.py

```diff
@@ -93,8 +93,38 @@ class TestAutomation(EditorTestSuite):
 
     # These tests require no UI interaction or modal dialog interactions
 
+    def cleanup_temp_levels(self, workspace):
+        file_system.delete([os.path.join(workspace.paths.engine_root(), "AutomatedTesting", "Levels", "tmp_level_1")],
+                        True, True)
+        file_system.delete([os.path.join(workspace.paths.engine_root(), "AutomatedTesting", "Levels", "tmp_level_2")],
+                        True, True)
+
     class test_EditorWorkflow_ParentEntityTransform_Affects_ChildEntityTransform(EditorBatchedTest):
         from .EditorScripts import EditorWorkflow_ParentEntityTransform_Affects_ChildEntityTransform as test_module
 
     class test_EditorWorkflow_ChildEntityTransform_Persists_After_ParentEntityTransform(EditorBatchedTest):
         from .EditorScripts import EditorWorkflow_ChildEntityTransform_Persists_After_ParentEntityTransform as test_module
+
+    class test_EditorWorkflow_EditorCameraMovesToEntityTransformWithBeThisCamera(EditorSingleTest):
```

Comment:
There are 6 new test files but only 5 added here. Missed to add the test `EditorWorkflow_EditorCameraGameModeTransitionWithMultipleCamerasReportsNoErrors` to this file?

Reply:
Oh good spot! 😬 I'll add that now, good eye thanks!